src/react-swipe.js

```diff
@@ -176,17 +176,33 @@ class ReactSwipe extends Component {
   }
 
   render() {
+    const { tagName,
+      className,
+      style,
+      children,
+      allowMouseEvents,
+      onSwipeUp,
+      onSwipeDown,
+      onSwipeLeft,
+      onSwipeRight,
+      onSwipeStart,
+      onSwipeMove,
+      onSwipeEnd,
+      ...props
+    } = this.props;
+
     return (
       <this.props.tagName
         ref={node => this.swiper = node}
         onMouseDown={ this._onMouseDown }
         onTouchStart={ this._handleSwipeStart }
         onTouchEnd={ this._handleSwipeEnd }
-        className={ this.props.className }
-        style={ this.props.style }
+        className={ className }
+        style={ style }
+        { ...props }
```

Comment:
You would probably want to apply move {...props} to before all other props are declared so required props are not overridden.

Reply:
I have declared all required props (except tolerance) above the `return`, so `props` variable contains only that properties, that is not used in this component.
So, there should be no collisions.

Reply:
@leandrowd Is all ok? If is, please, merge PR

Reply:
Merged and published to npm...